src/components/dateTimePicker/DateTimePicker.driver.ts

```diff
@@ -0,0 +1,31 @@
+import {fireEvent} from '@testing-library/react-native';
+import {
+  ComponentProps,
+  ExpandableOverlayDriver,
+  ButtonDriver
+} from '../../testkit';
+
+export const DateTimePickerDriver = (props: ComponentProps) => {
+  const {renderTree, testID} = props;
+  const expandableOverlayDriver = ExpandableOverlayDriver({...props, testID: `${testID}.overlay`});
+  const openPicker = () => {
+    expandableOverlayDriver.open();
+  };
+  const discardPicker = () => {
```

Comment:
Three options for names:
`discardAndClose` \ `cancel` \ `cancelAndClose`
In any case I don't think `picker` should be part of the name.

Reply:
Hmm. Not sure which one I think is better. I'll change it to close just to be aligned with the other drivers we have.